webui/react/src/components/UPlotChart/tooltipsPlugin.ts

```diff
@@ -0,0 +1,131 @@
+import uPlot, { Plugin } from 'uplot';
```

Comment:
**suggestion:** Would it make sense to filter out data where in a given `x` value is all series values are `null`? This more noticeable when there's only one series such as the screenshot below:
<img width="958" alt="Screen Shot 2021-04-19 at 5 39 25 PM" src="https://user-images.githubusercontent.com/220971/115317139-0c356b00-a138-11eb-93fc-a32eef2f1369.png">

So if the data input was something like this:
```
[
  [ 0, 100,  200,  300  ], # x-data
  [ 1, null, null, 1    ], # series 1
  [ 2, null, 2,    2    ], # series 2
  [ 3. null, 3,    null ], # series 3
]
```
where the first row is the X data and 2nd ~ 4th rows are series data, it would drop the 2nd column of each of the rows because the data for when X is 100 is all `null`. Basically making it:
```
[
  [ 0, 200,  300  ],
  [ 1, null, 1    ],
  [ 2, 2,    2    ],
  [ 3, 3,    null ],
]
```

Reply:
Consider that we're hiding series, so if you have a data array like this and the 1st serie is hidden, that would not work.

```
[
  [ 0, 100,  200,  300  ], # x-data
  [ 1, 1, null, 1    ], # series 1 (hidden)
  [ 2, null, 2,    2    ], # series 2
  [ 3. null, 3,    null ], # series 3
]
```

Possible alternative is to avoid displaying the toolltip and line if all the values are null?

Reply:
> Possible alternative is to avoid displaying the toolltip and line if all the values are null?

Thanks, this is a much better solution LOL, yes please do this if doable. 🙏